win32_compat/README

```diff
@@ -0,0 +1,28 @@
+# Windows compatibility layer
+
+This folder contains headeres and source code to make libsolv compile using MSVC
+in a Windows environment. Most of the sources contained in this folder were
+extracted from `musl`, a MIT licensed libc implementation. Below you find a 
+listing of all files, and their origin:
+
+- musl (MIT License)
+  - features
+  - fnmatch
+  - getopt
+  - regerror
+  - regex
+  - strcasecmp/strncasecmp
+  - strcasestr
+
+- stackoverflow (CC-BY-SA)
+  - unistd (https://stackoverflow.com/a/826027/1347553)
+  - pread (https://stackoverflow.com/a/42495892)
+
+- musl / TRE (BSD3 License)
```

Comment:
Please use SPDX notation for this particular license, as `BSD3` is ambiguous...

Reply:
makes sense.

Reply:
I've added the complete License in the `LICENSE` file in the win32 folder.